process_report/process_report.py

```diff
@@ -1,9 +1,68 @@
 import argparse
 import os
+import sys
 
+import json
 import pandas
 
 
+### Invoice field names
+INVOICE_DATE_FIELD = 'Invoice Month'
+PROJECT_FIELD = 'Project - Allocation'
+PROJECT_ID_FIELD = 'Project - Allocation ID'
+PI_FIELD = 'Manager (PI)'
+INVOICE_EMAIL_FIELD = 'Invoice Email'
+INVOICE_ADDRESS_FIELD = 'Invoice Address'
+INSTITUTION_FIELD = 'Institution'
+INSTITUTION_ID_FIELD = 'Institution - Specific Code'
+SU_HOURS_FIELD = 'SU Hours (GBhr or SUhr)'
+SU_TYPE_FIELD = 'SU Type'
+COST_FIELD = 'Cost'
+CREDIT_FIELD = 'Credit'
+CREDIT_CODE_FIELD = 'Credit Code'
+BALANCE_FIELD = 'Balance'
+###
+
+
+def get_institution_from_pi(pi_uname):
+    
+    dir_path = os.path.dirname(__file__)
+    with open(f'{dir_path}/institute_map.json', 'r') as f:
+        institute_map = json.load(f)
+
+    if '@' in pi_uname:
```

Comment:
This can be rewritten as the following, due to how split will still work regardless of whether there is `@` in the string, and how we always want the last entry in the list, accessible by `-1` index.

```python
institution_key = pi_uname.split('@')[-1]
institution = institute_map.get(institution_key, '')
```